tests/integration/kubernetes/k8s-confidential.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+# Copyright 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	# This check must be done with "<SPACE>${KATA_HYPERVISOR}<SPACE>" to avoid
+	# having substrings, like qemu, being matched with qemu-$something.
+	( [[ ! " ${SUPPORTED_HYPERVISORS[*]} " =~  " ${KATA_HYPERVISOR} " ]] ) && skip "Test not supported yet."
+
+	get_pod_config_dir
+	setup_unencrypted_confidential_pod
```

Comment:
This is going to run before every test in this bats. I understand that there's only one test right now, but it may be better to add this part to the `setup_file` method instead.